### PR TITLE
Currency answer component

### PR DESCRIPTION
**Pull request by bennpearson**
### What is the context of this PR?
Added currency answer component 

### How to review 

- Check component against trello task here: https://trello.com/c/asIXePnn/163-add-edit-currency-answer-component-3
- View in storybook
- Run tests

### CODE REVIEW

File: src/components/Answers/CurrencyAnswer/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      <SeamlessTextArea
+        cols="30"
+        rows="5"
+        placeholder="Enter a description (optional)…"
+        onChange={onChange}
+        value={answer.description}
+      />
+    </Field>
+    <Field id="answer.description">
+      <CurrencyIcon>£</CurrencyIcon>
```
Comment (samiwel) [inline src/components/Answers/CurrencyAnswer/index.js]: Just wondering if the Currency unit should be a prop and have a `defaultProp` of "£"?

**Comment by bennpearson** [comment]
[Add/edit currency answer component (3)](https://trello.com/c/asIXePnn/163-add-edit-currency-answer-component-3)

File: src/components/Answers/CurrencyAnswer/index.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+`;
+
+const CurrencyIcon = styled.span`
+  display: inline-block;
+  background-color: #f5f5f5;
+  border-right: 1px solid ${colors.borders};
+  border-radius: ${radius} 0 0 ${radius};
+  padding: 0.6em 0;
+  width: 2.5em;
+  font-weight: 600;
```
Comment (WickyNilliams) [inline src/components/Answers/CurrencyAnswer/index.js]: I don't think we have font-weight 600 available, looking at the CSS we have weights 400, 700, 900

File: src/components/Answers/CurrencyAnswer/index.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  font-size: 1em;
+  text-align: center;
+  line-height: normal;
```
Comment (WickyNilliams) [inline src/components/Answers/CurrencyAnswer/index.js]: Redundant?

File: src/components/Answers/CurrencyAnswer/index.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  position: absolute;
+  left: 0px;
```
Comment (WickyNilliams) [inline src/components/Answers/CurrencyAnswer/index.js]: Drop units here and below

File: src/components/Answers/CurrencyAnswer/CurrencyAnswer.story.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+import { action } from "@storybook/addon-actions";
+
+const Background = styled.div`
+  padding: 1em;
+  display: block;
+  max-width: 20em;
+`;
+
+const answer = {
+  title: "Lorem ipsum",
```
Comment (hamishtaplin) [inline src/components/Answers/CurrencyAnswer/CurrencyAnswer.story.js]: This should be `label`.

File: src/components/Answers/CurrencyAnswer/index.js [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    <Field id={`answers[${answerIndex}].description`}>
+      <SeamlessTextArea
+        cols="30"
+        rows="5"
+        placeholder="Enter a description (optional)…"
+        onChange={onChange}
+        value={answer.description}
+      />
+    </Field>
+    <Field id="answer.description">
```
Comment (hamishtaplin) [inline src/components/Answers/CurrencyAnswer/index.js]: You can remove this `Field` as it only exists to pass down the `id` to labels and inputs that need it—ie those that are wired up to the API. With the "dummy" fields, it's not needed. Add a `div` if you need a `block` context.

File: src/components/Answers/CurrencyAnswer/index.js [hunk 7]
```diff
@@ -52,7 +64,7 @@ const CurrencyAnswer = ({ answer, answerIndex, onChange }) =>
       />
     </Field>
     <Field id="answer.description">
-      <CurrencyIcon>£</CurrencyIcon>
+      <CurrencyComponent currencyUnit="£" />
```
Comment (WickyNilliams) [inline src/components/Answers/CurrencyAnswer/index.js]: no need to specify if it's the default

File: src/components/Forms/Field/index.js [hunk 8]
```diff
@@ -6,6 +6,7 @@ const StyledField = styled.div`
   display: block;
   width: 100%;
   margin-bottom: ${props => (props.last ? "0" : "1")}em;
+  position: relative;
```
Comment (hamishtaplin) [inline src/components/Forms/Field/index.js]: Is this needed?

**Pull request merged**